.github/labeler.yml

```
@@ -0,0 +1,27 @@
+#
+# Pull request label rules.
+#
+# Runs in action pull-request-labeler.yml.
+#
+# See https://github.com/marketplace/actions/labeler
+#
+
+# Labels for programming languages.
+javascript: ["**/*.js", "**/*.jsx"]
+typescript: ["**/*.ts", "**/*.tsx"]
+python: "**/*.py"
+terraform: "**/*.tf"
```

Quite a lot of .sh in the repo too, could we add it?

```suggestion
terraform: "**/*.tf"
shell: "**/*.sh"
```